docs/UserGuide.md

```diff
@@ -94,22 +94,20 @@ Shows a list of all persons in the address book.
 
 Format: `list`
 
-### Editing a person : `edit`
+### Editing an internship entry : `edit`
 
-Edits an existing person in the address book.
+Edits an internship entry from the list of existing entries.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Format: `edit INDEX [n/NAME] [r/ROLE] [s/STATUS ] [d/DATE]`
```

Comment:
In the `add` command, I wrote [n/COMPANY_NAME]. Do you want to change to COMPANY_NAME, or stick with NAME? I think both are fine but let's keep it consistent.

Reply:
Lets do COMPANY_NAME!